packages/camera/camera_android/example/integration_test/camera_test.dart

```diff
@@ -245,4 +245,44 @@ void main() {
       await controller.dispose();
     },
   );
+
+  testWidgets(
```

Comment:
@adam-harwood @stuartmorgan As a heads up, it looks like this test may be flaky on FTL. The roller succeeded pre-submit and then failed in post-submit. I couldn't reproduce the failure locally either.


```
12-09 14:12:29.267: I/flutter(23248): The following PlatformException was thrown running a test:
12-09 14:12:29.267: I/flutter(23248): PlatformException(error, stop failed., null, java.lang.RuntimeException: stop failed.
12-09 14:12:29.267: I/flutter(23248): 	at android.media.MediaRecorder.stop(Native Method)
12-09 14:12:29.267: I/flutter(23248): 	at io.flutter.plugins.camera.Camera.stopVideoRecording(Camera.java:779)
12-09 14:12:29.267: I/flutter(23248): 	at io.flutter.plugins.camera.MethodCallHandlerImpl.onMethodCall(MethodCallHandlerImpl.java:129)
12-09 14:12:29.267: I/flutter(23248): 	at
12-09 14:12:29.267: I/flutter(23248): io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
12-09 14:12:29.267: I/flutter(23248): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
12-09 14:12:29.267: I/flutter(23248): 	at
12-09 14:12:29.267: I/flutter(23248): io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
12-09 14:12:29.267: I/flutter(23248): 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
12-09 14:12:29.267: I/flutter(23248): 	at android.os.Handler.handleCallback(Handler.java:938)
12-09 14:12:29.267: I/flutter(23248): 	at android.os.Handler.dispatchMessage(Handler.java:99)
12-09 14:12:29.267: I/flutter(23248): 	at android.os.Looper.loop(Looper.java:223)
12-09 14:12:29.267: I/flutter(23248): 	at android.app.ActivityThread.main(ActivityThread.java:7664)
12-09 14:12:29.267: I/flutter(23248): 	at java.lang.reflect.Method.invoke(Native Method)
12-09 14:12:29.267: I/flutter(23248): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
12-09 14:12:29.267: I/flutter(23248): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
12-09 14:12:29.267: I/flutter(23248): )
12-09 14:12:29.267: I/flutter(23248): When the exception was thrown, this was the stack:
12-09 14:12:29.267: I/flutter(23248): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:653:7)
12-09 14:12:29.267: I/flutter(23248): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:296:18)
12-09 14:12:29.267: I/flutter(23248): <asynchronous suspension>
12-09 14:12:29.267: I/flutter(23248): #2      AndroidCamera.stopVideoRecording (package:camera_android/src/android_camera.dart:274:26)
12-09 14:12:29.267: I/flutter(23248): <asynchronous suspension>
12-09 14:12:29.267: I/flutter(23248): #3      CameraController.stopVideoRecording (package:camera_example/camera_controller.dart:330:9)
12-09 14:12:29.268: I/flutter(23248): <asynchronous suspension>
12-09 14:12:29.268: I/flutter(23248): #4      main.<anonymous closure> (file:///tmp/cirrus-ci-build/packages/camera/camera_android/example/integration_test/camera_test.dart:282:7)
12-09 14:12:29.268: I/flutter(23248): <asynchronous suspension>
12-09 14:12:29.268: I/flutter(23248): #5      testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:171:15)
12-09 14:12:29.268: I/flutter(23248): <asynchronous suspension>
12-09 14:12:29.268: I/flutter(23248): #6      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:841:5)
12-09 14:12:29.268: I/flutter(23248): <asynchronous suspension>
12-09 14:12:29.268: I/flutter(23248): The test description was:
12-09 14:12:29.268: I/flutter(23248):   recording with image stream
```

I'm not sure how flaky it is so far.

Reply:
Oh, that's what caused it to be referred before, but then it looked like it just always failed.

So probably quite flaky, and presubmit was lucky. We'll have to look at the test again next week.

Reply:
I'm not really sure how to debug this. Looks like it's green for everyone locally. Is there more context on the stack trace that is being omitted? From that trace I can't even see where the call to `stopVideoRecording` is coming from.

Reply:
I'm trying https://github.com/flutter/plugins/pull/6831 to see how that goes.

Reply:
It's looking like that was it, I don't see this flake in the commits since that landed.

Reply:
Thanks for figuring this out @stuartmorgan !